ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activitySection.component.ts

```diff
@@ -90,7 +92,8 @@ export class ActivitySectionComponent {
     @Input() public activityGuid: string;
     @Output() visibilityChanged: EventEmitter<BlockVisibility[]> = new EventEmitter();
     @Output() valueChanged: EventEmitter<AnswerValue> = new EventEmitter();
-    @Output() embeddedComponentValidStatusChanged: EventEmitter<boolean> = new EventEmitter();
+    @Output() addressComponentValidStatusChanged: EventEmitter<boolean> = new EventEmitter();
+    @Output() institutionComponentValidationStatusChanged: EventEmitter<boolean> = new EventEmitter();
```

Comment:
I would consider consolidating `addressComponentValidStatusChanged` and `institutionComponentValidationStatusChanged` into just one output variable (maybe bringing back `embeddedComponentValidStatusChanged`) and then either changing the type to EventEmitter<boolean[]> or calculating an EventEmitter<boolean> locally using a `reduce` operator.
I say this because having the specialized output variables:
1) Leaks to the component user what is in the section. Was not happening before.
2) The user of `section` is going to use the two variables in exactly the same way; does not matter if one is address and the other institution.